### DISCUSSION TITLE
When uploading large media file from UI, broadcasting server hangs and player stops

### DISCUSSION THREAD

**Discussion by gmantzak** [AzuraCast/AzuraCast#5862] [Q&A] [answered]
Hi Everybody :)

I have a problem, when uploading a large (>100mb) media file , either podcast or music the server hangs and the web player stops 
until the file uploading has finished. So , for music I use ftp which doesn't affect it. Have you noticed something similar?
I don't know if it's the machine , but a 4 CPU 8GB it should be ok in theory.

Thank you
Georgios

**Comment by Moonbase59** [comment, marked as answer]
Firefox, for example, is notorious for hanging sometimes. So if you have the web player open in another tab _in the same browser_, this can happen.

Check if the stream _really_ hangs by using an audio player pointed at the stream, or a second browser.

**Comment by scysys** [comment]
> I have a problem, when uploading a large (>100mb) media file , either podcast or music the server hangs and the web player stops
> until the file uploading has finished.

That's a normal behavior that can happen in multiple scenarios. In most cases, you can't open another page or tab from the same domain until the process is finished. It happens just for you, not for visitors or anyone else.

Like @Moonbase59 said. Just open another browser if you cant wait the upload is finished.

**Reply by gmantzak** [reply]
Thank you very much both!
